docs/community/maintainers-guide/releasing.rst

```diff
@@ -0,0 +1,224 @@
+      documented, to help users when upgrading.
+3. Prepare release notes (`see example <https://github.com/inveniosoftware/invenio/commit/f4d0aa5ac78d76228fe86754eeb3bbfe81a1854f>`_):
+    - In ``docs/releases/``, copy an existing patch-level or minor
+      release notes (e.g. ``docs/releases/v3.1.2.rst``).
+    - Edit release notes.
+    - Include the new release notes into ``docs/releases/index.rst``.
+    - Check the "Maintenance policy", e.g. is the version correct?
+      (`example <https://github.com/inveniosoftware/invenio/commit/edb863d2f5228fb158c090a69c2db7c3385b6ba3>`_).
+4. Create a release commit (message pattern:
+   ``release: v<major>.<minor>.<patch>``).
+5. Create tag and push release:
+    - ``git tag v<major>.<minor>.<patch>``
+    - ``git push inveniosoftware master v<major>.<minor>.<patch>``
+6. Regularly check ``Travis`` and ``PyPI`` to ensure the release went through.
+
+Manual releases
+---------------
+
+When the process of releasing fails for some reasons, you might want to
+manually publish the new version of a package.
+
+**PyPI**
+
+You can manually reproduce the publishing process done by ``Travis`` by doing:
+
+1. Activate your virtualenv for the package that you want to release.
+2. Generate the different distributions:
+    - ``python setup.py compile_catalog sdist bdist_wheel``
+3. Install the tool to upload releases to PyPI:
+    - ``pip install twine wheel``
+4. Publish:
+    - ``twine upload --repository <repo-name> dist/*``. The command will
+      ask for username and password. Architects should have the credentials.
+
+**NpmJS**
+
+Manual release on ``NpmJS`` is not only needed in case of failure, but also
+when creating a totally new package, never released yet.
+
+1. Make sure that you have an updated version of the
+   `npm <https://www.npmjs.com/get-npm>`_ client installed in your machine.
+2. Login on ``NpmJS``:
+    - ``npm login``: you will need ``inveniosoftware`` username and password.
+      Architects should have the credentials.
+3. Optionally, verify the package before releasing:
+    - ``npm pack``: this will create the final archive that will be
+      published on ``NpmJS`` in case you want to check its content.
+4. Understand if the package that you are publising is
+   ``scoped`` (``@inveniosoftware``) or not. For example, ``react-searchkit``
+   is not scoped, but ``@inveniosoftware/react-invenio-app-ils`` is scoped.
+5. Publish:
+    - `not scoped`: ``npm publish --dry-run`` to double check that everything
+      is ok, then ``npm publish``.
+    - `scoped`: you need to add the ``access`` param to publish it as public,
+      otherwise by default, it will be ``restricted``.
+      Run ``npm publish --access public --dry-run`` to double check that
+      everything is ok, then ``npm publish --access public``.
+
+
+Announcing release
+==================
+
+You should announce a new Invenio release in the following channels:
+
+- Invenio-Talk Announcement
```

Comment:
What do you think of adding links to examples of all of these communications? so we save adding a template here

Reply:
We can add pointers, templates do not exists per se (IIRC). E.g. the announcement might differ a lot depending on the release. What about:

- https://inveniosoftware.org/blog/invenio-v33-released/
- https://invenio-talk.web.cern.ch/t/invenio-v3-2-released/62/3
- https://discordapp.com/channels/692989811736182844/704628216857362473/740206015957434439

There is only announcement of RDM ones in discord... Maybe we can wait till v3.4 is out and put all together the same?

Reply:
Links added but Discord!